src/types/objectMaps.ts

```diff
@@ -0,0 +1,20 @@
+import { FilterByOptions, FilterOptionsKeywords, IAssigneeCombined } from '@/types/interfaces'
+
+export const filterTypeToButtonIndexMap: Record<string, number> = {
+  [FilterOptionsKeywords.CLIENTS]: 2,
+  [FilterOptionsKeywords.TEAM]: 1,
+  '': 3,
```

Comment:
This can be interpreted as `default` just like below? It is creating a bit of confusion because same entity is defined as `default` and `null`.